src/vs

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
Can you `chmod 755` this file so it works under cygwin as well?

Reply:
Yes, sorry about that! I forgot that the MSYS implementation doesn't preserve file attributes.